### DISCUSSION TITLE
Insomnia REST Client - Cannot set Content Type in multipart Form

### DISCUSSION THREAD

**Discussion by soubhikchatterjee** [Kong/insomnia#6901] [Ideas]
When making an API request for "form-data", I need to set content-type application/json for the "mailInfo" field (shown in the screenshot below) in Insomnia. I can toggle this option in Postman but I don't seem to find a similar option in Insomnia.

I am using Insomnia 8.4.2

Please help!

![image](https://github.com/Kong/insomnia/assets/863672/07ed4511-d783-4d2f-b6cf-b198e5feef5f)

**Comment by ash-eng-spin** [comment]
Has this ever been updated? I'm on Insomnia 8.6.0 and do not see the ability for the user to edit content-types for the parts in a multi-part request. This is a show stopper for us.

**Comment by ash-eng-spin** [comment]
Wanted to add that users should be able to type whatever they want in content-type whether it's a text part or a file part.

**Text:** user should be able to set because this could be blank, application/json, application/xml, text/plain or many other things.
**Files:** user should be able to set because this could be blank, application/pdf, application/octet-stream, application/vnd.ms-excel, text/csv or many other things.

**Comment by subnetmarco** [comment]
It is now possible since Insomnia 9.x to remove the `content-type` header, or set it to a different value.

**Comment by nelosnlx** [comment]
i find the same question , do you have solved the problem? my Version is 12.1.0
